lib/BraveRewards.framework/Headers/ledger.mojom.objc.h

```diff
@@ -61,11 +61,27 @@ typedef NS_ENUM(NSInteger, BATResult) {
   BATResultPendingNotEnoughFunds = 22,
   BATResultRecurringTableEmpty = 23,
   BATResultExpiredToken = 24,
+  BATResultBatNotAllowed = 25,
+  BATResultAlreadyExists = 26,
 } NS_SWIFT_NAME(Result);
 
 
+typedef NS_ENUM(NSInteger, BATPublisherStatus) {
+  BATPublisherStatusNotVerified = 0,
+  BATPublisherStatusConnected = 1,
+  BATPublisherStatusVerified = 2,
+} NS_SWIFT_NAME(PublisherStatus);
 
-@class BATContributionInfo, BATPublisherInfo, BATPublisherBanner, BATPendingContribution, BATPendingContributionInfo, BATVisitData, BATGrant, BATWalletProperties, BATBalance, BATAutoContributeProps, BATMediaEventInfo, BATExternalWallet, BATBalanceReportInfo, BATActivityInfoFilterOrderPair, BATActivityInfoFilter, BATReconcileInfo, BATRewardsInternalsInfo;
+
+typedef NS_ENUM(NSInteger, BATRewardsCategory) {
+  BATRewardsCategoryAutoContribute = 2,
+  BATRewardsCategoryOneTimeTip = 8,
+  BATRewardsCategoryRecurringTip = 16,
```

Comment:
Would be much better to use bit-shifting to be more explicit that the enum values can be "OR'd" than hardcoding the result of it.

IE:

```
1 << 1  = 2
1 << 2 = 4
1 << 3  = 8
1 << 4 = 12
1 << 5  = 16
```

and so on.

Reply:
This is autogenerated based on this mojom file: https://github.com/brave/brave-core/blob/4d2e01bc756cc5c76a8444509c4182cbc47e3c35/vendor/bat-native-ledger/include/bat/ledger/public/interfaces/ledger.mojom#L248

We aren't defining these values. I can only assume they started out like that and options were removed or something